src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -116,11 +123,15 @@
 
     @Shadow private String translator;
     @Shadow public NetHandlerPlayServer playerNetServerHandler;
+    @Shadow public MinecraftServer mcServer;
     @Shadow public ItemInWorldManager theItemInWorldManager;
     @Shadow public int lastExperience;
-    @Shadow public MinecraftServer mcServer;
     @Shadow public abstract void setSpectatingEntity(Entity entityToSpectate);
     @Shadow public abstract void sendPlayerAbilities();
+    @Shadow private EntityPlayer.EnumChatVisibility chatVisibility;
+    @Shadow private boolean chatColours;
+    private Set<SkinPart> skinParts;
```

Comment:
Make this a empty set or a set containing all parts to prevent returing null in the getter if called too early.